frontend/src/pages/workspace/profile/Index.tsx

```diff
@@ -0,0 +1,111 @@
+import { Container, Stack, Avatar, Typography, Button, FormControl } from "@mui/material";
+import { TextFieldElement, FormContainer } from "react-hook-form-mui";
+import { useSelector } from "react-redux";
+import { selectUser } from "../../../store/userSlice";
+import { useEffect, useMemo, useState } from "react";
+import { useDebounce } from "react-use";
+import { useUpdateUserNicknmaeMutation } from "../../../hooks/api/user";
+import { useCheckNameConflictQuery } from "../../../hooks/api/check";
+import { useGetWorkspaceQuery } from "../../../hooks/api/workspace";
+import { useParams } from "react-router-dom";
+const avatarSize = 117;
+
+function ProfileIndex() {
+	const params = useParams();
+	// eslint-disable-next-line
+	const { data: workspace } = useGetWorkspaceQuery(params.workspaceSlug);
+
+	const userStore = useSelector(selectUser);
+	const [nickname, setNickname] = useState(userStore.data?.nickname || "");
+	const [debouncedNickname, setDebouncedNickname] = useState("");
+	const { data: conflictResult } = useCheckNameConflictQuery(debouncedNickname);
+	const { mutateAsync: updateUserNickname } = useUpdateUserNicknmaeMutation();
+	const errorMessage = useMemo(() => {
+		if (conflictResult?.conflict) {
```

Comment:
When the nickname doesn't changed, we can use this code.

```typescript
if (debouncedNickname != userStore.data?.nickname && conflictResult?.conflict) {
    // ...
}
```

Or, you can add disable option in `useCheckNameConflictQuery`.

Reply:
thank you i add more condition [16ffca1](https://github.com/yorkie-team/codepair/pull/237/commits/16ffca1356e8db81e7bb1866716335c1f4d6f250)